### DISCUSSION TITLE
The use of CellRendererComponent to get the position of the elements

### DISCUSSION THREAD

**Discussion by tsnery** [Shopify/flash-list#515] [Q&A]
Hi, I'm currently using FlashList and I have a question about how to get the position of each element inside the list. When using FlatList I just used to create a CustomRendererComponent to get it, but I cannot get reproduce the same way in FlashList.

**Comment by naqvitalha** [comment]
You can also get it from `flashListRef.recyclerlistview_unsafe.getLayout(index)`
